systems/plants/@RigidBodyManipulator/doKinematics.m

```diff
@@ -1,4 +1,4 @@
-function kinsol=doKinematics(model,q,b_compute_second_derivatives,use_mex,qd)
+function kinsol=doKinematics(model,q,b_compute_second_derivatives,use_mex,qd,options)
```

Comment:
I agree. I get the order of the boolean arguments wrong almost as often as I get it right.